scripts/LAUNCH.sh

```diff
@@ -1,27 +1,27 @@
 #!/bin/bash
 # File:  LAUNCH.sh
 # Date:  28-Jun-2020
-#
-# Example uvicorn deployment on non-privileged port.
-#
-#   nohup ./scripts/LAUNCH.sh >& LOGLAUNCH
-##
-HERE="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
-TOPDIR="$(dirname "$HERE")"
-OE_LICENSE=${OE_LICENSE:=~/oe_license.txt}
```

Comment:
So the app is able to launch even without the OE_LICENSE variable being set? Or is it being set somewhere else?

Reply:
The changes to the `LAUNCH.sh` scripts were done by Connor, so I'm not sure if this is by design. @zkzfng did you have a chance to review `LAUNCH.sh` script with Connor already?

Reply:
Zukang replied by to my email directly; looks like the changes were made to be in line with the launch script in another repo (https://github.com/rcsb/py-rcsb_app_file/blob/master/deploy/launch.sh).

Reply:
Ah, I see. I think he might have been slightly confused in this case, because the chem app requires the OE license (as well as some of the other env variables that he deleted from this script). I would recommend reverting back to the original script...

Reply:
Sounds good, the script has been reverted back to the original.